frontend/src/components/ui/AppDocumentUpload.vue

```diff
@@ -2,12 +2,12 @@
   <v-container fluid class="pa-0">
```

Comment:
I'm getting a couple eslint errors in this file (lines 23 and 39) that it might be good to cleanup.

Reply:
it's updated